files/Aboot/boot0.j2

```diff
@@ -24,7 +24,7 @@ kernel_params=kernel-params
 
 aboot_machine="arista_unknown"
 
-target_path=/mnt/flash
+[ -z "$target_path" ] && target_path=/mnt/flash
```

Comment:
Is it more readable to rewrite similar blocks by 'if-else'.
The shortcuts is useful for simple error handling, but confusing for complex conditions.